services/learn-card-network/brain-service/src/helpers/notifications.helpers.ts

```diff
@@ -0,0 +1,97 @@
+interface IPushNotificationRequest {
+    to: string;
+    title: string;
+    message?: string;
+    url?: string;
+    actionType: string;
+}
+
+interface IPushRegistrationRequest {
+    profileId: string;
+    deviceToken: string;
+}
+
+export async function SendPushNotification(args: IPushNotificationRequest) {
+    try {
+        if (process.env.WEBHOOK_URL) {
```

Comment:
Have we set this in git secrets? If not, we should!

Should we rename it to NOTIFICATIONS_SERVICE_WEBHOOK_URL to disambiguate?

Reply:
changed